app/helpers/tasks_helper.rb

```diff
@@ -0,0 +1,42 @@
+module TasksHelper
+  TASKS_LIMIT = 5
```

Comment:
Even if we want to hard code, and not make this configurable, a Helper seems to be the wrong place for this constant?

Perhaps the `Configuration` class?

Reply:
this was just for presentation, but more geared towards the pro dashboard for consistency there. I have removed this from CE since it's introducing an inconsistency in the project-level dashboard.